### ISSUE TITLE
Release v1.3.0

### ISSUE THREAD

**Issue by AirQuick** [xspec/xspec#563]
@cirulls
I think we're getting close to releasing v1.3.0 because

* The outstanding pull requests #459 or later are mainly for XQuery. (#458 contains a regression bug fix and should be included in v1.3.0.)
* The draft release note looks long enough now :)

Thanks for compiling and maintaining the release note which is very useful for looking back the changes. The highlights would be:

* Various bugs in the report files have been fixed (#177, #355, #367 etc.)
* Handling whitespace-only text nodes has been fixed and [documented](https://github.com/xspec/xspec/wiki/Writing-Scenarios#whitespace-only-text-nodes) (#346 etc.)
* Testing Schematron with text nodes has been improved and [documented](https://github.com/xspec/xspec/wiki/Testing-Schematron-with-Text-Nodes) (#396 etc.)
* [Code Coverage](https://github.com/xspec/xspec/wiki/Code-Coverage) has been stabilized (#214, #354 etc.)
* Some XQuery bugs have been fixed (#440, #441 etc. The other known bugs will be fixed in the next release.)

To track the release workflow, I copied some items from [Release Workflow](https://github.com/xspec/xspec/wiki/Release-Workflow) to [the project board](https://github.com/xspec/xspec/projects/4).

I've put the rest of the outstanding pull requests to [v1.4.0](https://github.com/xspec/xspec/projects/5).

**Comment by cirulls** [comment]
@AirQuick: thanks for your amazing work, this is awesome! 

I understand #458 is the last pull request to be merged into `master`. I will use your comments for the highlights and may ask you to double check before I push the publish release button. I should be able to do all the tasks necessary for the release during this weekend (4-5 May). Hopefully this time I will correctly deploy the XSpec artifact to Maven Central (and fix #506 too).

**Comment by cirulls** [comment]
@AirQuick: Please have a look at the draft release notes for v1.3.0 and let me know if they look fine to you. I created #567 to increase the version in the pom file, I'm just waiting for the test suite to complete successfully before merging it. 

If you confirm that the release notes are fine, I'll push the release button straight after #567 is merged into `master`.

**Comment by AirQuick** [comment]
Thanks, @cirulls 
Both the release note and #567 look good to me.

**Comment by cirulls** [comment]
@AirQuick: thanks for confirming. I'm merging the last pull request and making the v1.3.0 release.

**Comment by cirulls** [comment]
Following the [release workflow](https://github.com/xspec/xspec/wiki/Release-Workflow)  I did this:

✔️ Post a message on the XSpec mailing list
✔️ Promote the new release on xml.com
✔️ Post a message on the xml mailing list
✔️ Ask users to tweet about the new release: @cmarchand and @yamahito, I know you are quite active on Twitter, could you please tweet about the latest XSpec v1.3.0 release available at https://github.com/xspec/xspec/releases/latest
✔️ Create a pull request with `<version>1.4.0-SNAPSHOT</version>` in `pom.xml` to start development of the next XSpec release (#568).

Still outstanding:

- Deploy the XSpec artifact to Maven Central: I'm planning to give it a go at this in the next days, probably on Thursday 9th May.

**Comment by cmarchand** [comment]
Sonatype has changed its requirements. `pom.xml` must be updated.  
I'm working on this and will send a PR for this.  
I'll probably deploy the 1.3.0 release to maven central to check that `pom.xml` is Ok.

**Comment by cirulls** [comment]
@cmarchand: that would be great (and less work for me :wink:). In the pull request could you please also mention what has changed in the Sonatype requirements so that we can update [this documentation](https://github.com/xspec/xspec/wiki/Deploy-XSpec-artifact-to-Maven-Central)?

**Comment by cmarchand** [comment]
I'm not sure if Sonatype has changed something. But I'm sure we are not able to deploy with the actual pom and configuration.

**Comment by cmarchand** [comment]
Javadoc was not generated, ad soatype requires sources & javadoc for ope-source projects.
Solved by https://github.com/xspec/xspec/pull/569, ad tested by deployig 1.3.0

**Comment by AirQuick** [comment]
Maybe for the future releases, we need to swap these steps?
> 3. Publish the release
> 4. Deploy the XSpec artifact to Maven Central following these instructions

**Comment by cirulls** [comment]
I swapped the steps in the release workflow (see [here](https://github.com/xspec/xspec/wiki/Release-Workflow/_compare/e661d8d0c04c1e6e2dccefc5b66f97d22a8074aa...9f705c72e63280d29406acd3b55c1b78e3b73a1c)).

**Comment by cirulls** [comment]
@AirQuick and @cmarchand: I merged #569 and updated the wiki. I think this issue can be closed now. Is that fine for you?

**Comment by AirQuick** [comment]
👍

**Comment by AirQuick** [comment]
Thanks, @cmarchand and @cirulls 
I'm closing this and moving on to developing the next version.
